src/04-modality-specific-files/08-genetic-descriptor.md

```diff
@@ -0,0 +1,104 @@
+| Genetics.Database    | OPTIONAL. URI of database where the dataset is hosted.                         |
+| Genetics.Descriptors | OPTIONAL. List of relevant descriptors (*e.g.*, journal articles) for dataset. |
+
+Example:
+
+```JSON
+{
+  "Name": "Human Connectome Project",
+  "BIDSVersion":  "1.3.0",
+  "License": "CC0",
+  "Authors": ["1st author", "2nd author"],
+  "Funding": ["P41 EB015894/EB/NIBIB NIH HHS/United States"],
+  "Genetics": {
+     "Dataset": "https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs001364.v1.p1",
+     "Database": "https://www.ncbi.nlm.nih.gov/gap/",
+     "Descriptors": ["https://doi.org/10.1016/j.neuroimage.2013.05.041"]
+     }
+}
+```
+
+## Subject naming and Participants file
+
+If the same participants have different identifiers in the genetic and imaging datasets,
+the column `GeneticID` SHOULD be added to the `participants.tsv` file to associate
+the BIDS participant with a subject in the `Genetics.Dataset` referred to in the
+`dataset_description.json` file.
+
+Information about the presence/absence of specific genetic markers MAY be duplicated
+in the `participants.tsv` file by adding optional columns.
+
+`participants.tsv` example:
+
+```Text
+participant_id	age	sex	group	genetic_id	idh_mutation
+sub-control01	34	M	control	124587	yes
+sub-control02	12	F	control	548936	yes
+sub-patient01	33	F	patient	489634	no
+```
+
+## Genetic Information
+
+Template:
+
+```Text
+genetic_info.json
+```
+
+The `genetic_info.json` file describes the genetic information available in the
+`participants.tsv` file and/or the genetic database described in
+`dataset_description.json`.
+Datasets containing the `Genetics` field in `dataset_description.json` or the
+`genetic_id` column in `participants.tsv` MUST include this file with the following
+fields:
+
+| Field name         | Definition                                                                  | Values                                                                                                                                                                       |
+| :----------------- | :-------------------------------------------------------------------------- | :--------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| GeneticLevel       | MANDATORY Describes the level of analysis                                   | `Genetic`, `Genomic`, `Epigenomic`, `Transcriptomic`, `Metabolomic`, or `Proteomic`                                                                                          |
+| AnalyticalApproach | OPTIONAL Methodology or methodologies used to analyse the GeneticLevel      | String or list of strings. Each Value must be taken from [gapsolr][] under /Study/Molecular Data Type, for instance `SNP Genotypes (Array)` or `Methylation (CpG)`           |
```

Comment:
Is [gapsolr][] supposed to link to another resource ?

Reply:
As is, just the shortened link is appearing as the name and it seems a bit confusing ! But maybe that was intentional ?

Reply:
nope it was an oversight that you fixed :-) thx